frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/LookAndFeelSettings.tsx

```diff
@@ -1,9 +1,8 @@
-import type { ChangeEvent } from "react";
+import { match } from "ts-pattern";
```

Comment:
All changes are to match the new design minus the move of transparent theme to its toggle, that will be another PR.